src/App.vue

```diff
@@ -401,7 +410,25 @@ export default Vue.extend({
                 this.appStore.setStateFromJSONStr( 
                     {
                         stateJSONStr: savedState,
-                        callBack: () => {},
+                        callBack: (loadedSuccesfully) => {
+                            // When a file had been reloaded and it was previously synced with Google Drive, we want to ask the user
+                            // about reloading the project from Google Drive again (if the file changed on Google Drive side)
```

Comment:
I think this is just an issue with the comment and not the code, but I think the bracket is not right: we always want to ask even if the project didn't change in Google Drive because it's about setting up the syncing as much it is about reloading the changed state.

Reply:
The comment didn't mean to say we ask only if the file has changed. I wanted to say "the file could have changed". But that's true that even if that one reason, it's also just about doing the connection with Google Drive as you mentioned. I'll remove it.